sql/moz-fx-data-shared-prod/mozilla_vpn_derived/all_subscriptions_v1/query.sql

```diff
@@ -494,6 +488,13 @@ vpn_subscriptions AS (
     *
   FROM
     android_iap_subscriptions
+),
+vpn_subscriptions_with_end_date AS (
+  SELECT
+    *,
+    COALESCE(ended_at, TIMESTAMP(CURRENT_DATE)) AS end_date,
```

Comment:
`all_subscriptions_v1` is updated to consistently define `ended_at` as always `NULL` if it's in the future, and `end_date` as never `NULL`, so that aggregate views don't depend on `CURRENT_DATE`.